src/scripts/run_full_dialogue_pipeline.js

```
@@ -1,221 +1,235 @@
+        break
+
       case '4':
         // Run full pipeline (batch generate + validate)
-        console.log('\n=== Step 1: Batch Generate Dialogues ===\n');
-        
+        console.log('\n=== Step 1: Batch Generate Dialogues ===\n')
+
         // Run batch generation (options handled by the script itself)
-        const batchSuccess = await runScript(BATCH_GENERATE_SCRIPT);
-        
+        const batchSuccess = await runScript(BATCH_GENERATE_SCRIPT)
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)